src/app/my-rewards/utils.ts

```diff
@@ -0,0 +1,16 @@
+import { formatSymbol } from '@/app/collective-rewards/rewards'
+import { getFiatAmount } from '@/app/collective-rewards/rewards'
+import { formatCurrency } from '@/lib/utils'
+import { USD } from '@/lib/constants'
+
+export const formatRewards = (rewards: bigint, price: number, symbol: string) => {
```

Comment:
I think that we could propagate this or even replace using our previous formatter.

Reply:
There is an ongoing conversation about it and I think we'll need to review everything before we deliver, as there is some inconsistencies at the moment